bridges/snowbridge/pallets/inbound-queue-v2/src/mock.rs

```diff
@@ -285,9 +285,13 @@ pub fn mock_execution_proof() -> ExecutionProof {
 // For backwards compatibility and tests
 impl WeightInfo for () {
 	fn submit() -> Weight {
-		Weight::from_parts(70_000_000, 0)
-			.saturating_add(Weight::from_parts(0, 3601))
-			.saturating_add(RocksDbWeight::get().reads(2))
+		// Proof Size summary in bytes:
+		//  Measured:  `309`
+		//  Estimated: `3774`
+		// Minimum execution time: 59_000_000 picoseconds.
+		Weight::from_parts(60_000_000, 0)
+			.saturating_add(Weight::from_parts(0, 3774))
+			.saturating_add(RocksDbWeight::get().reads(7))
```

Comment:
Weights added here instead of in `weights.rs` because of the comment: https://github.com/paritytech/polkadot-sdk/pull/6697#discussion_r1935505229

Reply:
I'm not sure moving it out is nessessary. I'd assume the Weights in `weight.rs` is just some default values for mock/tests. That's what most of the exist polkadot-sdk pallets does.
e.g. 
https://github.com/paritytech/polkadot-sdk/blob/ce5f89c6986256e898123dd37ccc7d12de512317/substrate/frame/system/src/weights.rs#L88

The weight in the runtime is what needs to be benchmarked, but I'm not too worried about it at the moment. I'd assume there is some CI flow where Parity DevOps will generate the weights on a recommended machine before release. Runing benchmark on our dev PC may not be a good practise.

@vgeddes can correct me if I'm wrong.

Reply:
Yeah I don't understand Adrian's comment. The convention is that each pallet has a `weights.rs` for use by tests. I see this throughout polkadot-sdk.

Reply:
Moved weights back: https://github.com/vgeddes/polkadot-sdk/pull/33/commits/c3905d72d3e65fbbda33716053f85b66f65edace